magnum_cluster_api/driver.py

```diff
@@ -45,6 +45,9 @@ def create_cluster(self, context, cluster, cluster_create_timeout):
             self.k8s_api, cluster
         ).apply()
 
+        if utils.get_cluster_label_as_bool(cluster, "auto_scaling_enabled", False):
+            resources.ClusterAutoscalerHelmRelease(self.k8s_api, cluster).apply()
```

Comment:
let's move this to `resources.apply_cluster_from_magnum_cluster`, after creating the `HelmRepository`.